blake2s.d.ts

```diff
@@ -0,0 +1,22 @@
+export as namespace BLAKE2s;
+
+export type ByteArray = number[] | Uint8Array;
+
+export default class BLAKE2s {
+  constructor(digestLength: number | undefined, keyOrConfig: ByteArray | BLAKE2sConfig);
```

Comment:
`keyOrConfig` is optional.

Can a type definition contain multiple declarations for function that accepts different arguments, like normal TypeScript?

```
constructor(digestLength: number?, key?: ByteArray);
constructor(digestLength: number?, config?: BLAKE2sConfig);
```

I wonder if it also can specify the default value as `digestLength = 32`?

Reply:
Fixed with an overloaded constructor. You can't provide a parameter initializer in defs, unfortunately.